js/mapfilter/config.js

```diff
@@ -54,6 +54,45 @@ module.exports = require('backbone').Model.extend({
       return [lat, lng];
     else
       return [this.defaults.mapCenterLat, this.defaults.mapCenterLong];
+  },
+
+  /*
+   * Should be an object with the following properties:
+   *
+   * tiles: {
+   *   url: '/tileLayers', // the url to get the list of layers
+   *   tilesPath: '/monitoring-files/Maps/Tiles' // the base path for tiles
+   * }
+   */
+  getTilesInfo: function() {
```

Comment:
This gets populated via /json/mapfilter-config.json.  From the monitoring station's `controllers/map-filter.js`, we drive what gets populated here.

In the other PR, we always pass this object, so this always has data in that context.  The web version won't (but, of course, we can if we add that as a feature).